build.gradle

```diff
@@ -46,7 +46,6 @@ dependencies {
 	// Integration tests - test containers dependencies
     testImplementation "org.testcontainers:testcontainers:$testcontainersVersion"
     testImplementation "org.testcontainers:mongodb:$testcontainersVersion"
-	testImplementation 'org.mongodb:mongodb-driver-sync:5.0.1'
```

Comment:
This conflicts with the version of mongo driver brought in by `spring-boot-starter-data-mongodb`.